modules/acs-requirements.adoc

```diff
@@ -36,3 +36,9 @@ However, you can use another storage type if you do not have SSDs available.
 Use the `helm version` command to verify the version of Helm you have installed.
 * The {ocp} CLI (`oc`).
 * You must have the required permissions to configure deployments in the Central cluster.
+* You must have access to the Red Hat Container Registry. See link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication] for information about downloading images from `registry.redhat.io`.
++
+[NOTE]
+====
+If you are not a Red Hat customer and purchased the Stackrox Kubernetes Security Platform before the acquisition. You can use StackRox’s container registry at `stackrox.io`. Contact mailto:support@stackrox.com[support@stackrox.com] to enable access to the registry.
```

Comment:
First sentence is a frag, right?

If you are not a Red Hat customer and purchased the Stackrox Kubernetes Security Platform before the acquisition, you can use the Stackrox container registry at `stackrox.io`.

Reply:
Ah 🦊 fixed it. Thanks.